scripts/exprep_post.sh

```diff
@@ -310,6 +389,212 @@ fi
 
 fi # test for PROCESS_REMOREST=YES
 
+# PROCESS_REMOREST_dm2 = YES runs only on demand, on dev machines
+# Unrestrict 48h old aircraft obs(exc. TAMDAR), add *.ur file to $COMOUTm2
+dev_m=$(grep backup /lfs/h1/ops/prod/config/prodmachinefile | cut -d: -f2)
+this_m=$(cat /etc/cluster_name)
+
+if [ "$PROCESS_REMOREST_dm2" = 'YES' -a $this_m = $dev_m ]; then
+  cdate10M2=`$NDATE -$tmhr $PDYm2$cyc`
```

Comment:
Will this job only be run on WCOSS2 or should it also be able to run on the RDHPCS machines?  If the latter, then a different check should be used.  If the former, then the comment should be clarified that this job will only run on the develop WCOSS2 cluster.

Reply:
I see you point here - the check doesn't consider the RDHPCS machines.

This job should only run on develop WCOSS2 cluster **or** on RDHPCS. 
Never in production! (because we write into previous days directories and NCO isn't allowing it) .

Reply:
We don't currently need to run this job in development runs of the GFS and we also discontinued archiving unrestricted versions of the restricted dump files...so I don't think we need to worry about this section in terms of the GFS using it in development runs outside of ops.